src/chainparams.cpp

```diff
@@ -159,6 +191,23 @@ class CMainParams : public CChainParams
     {
         networkID = CBaseChainParams::MAIN;
         strNetworkID = "main";
+
+        consensus.BIP65Height = 1808634; // 82629b7a9978f5c7ea3f70a12db92633a7d2e436711500db28b97efd48b1e527
+        consensus.powLimit   = uint256S("00000ffff0000000000000000000000000000000000000000000000000000000");
+        consensus.posLimitv1 = uint256S("000000ffff000000000000000000000000000000000000000000000000000000");
+        consensus.posLimitv2 = uint256S("00000ffff0000000000000000000000000000000000000000000000000000000");
+        consensus.nCoinbaseMaturity = 100;
+        consensus.nTargetTimespan = 40 * 60;
+        consensus.nTargetSpacing = 1 * 60;
```

Comment:
Same here, `consensus.nTargetTimespan` and `consensus.nTargetSpacing` are repeated. You didn't remove the chainparams fields.

Reply:
Have begun migration where these instances are referred to in https://github.com/barrystyle/PIVX/tree/022020-rebase-consensuscorrections